kitchen/modules/child-trie/src/lib.rs

```diff
@@ -1 +1,149 @@
-// child trie storage example
\ No newline at end of file
+/// Child Trie Storage Example (designed to be minimal and generic)
+/// - for efficient storage of (key, value) pairs when **proofs of key inclusion** need to be cheap
+///
+/// - the root of the child trie is used to prove participation of the `key`
+/// - the trie is retained for efficient usage of the associated `value` later
+use support::{
+	decl_module, decl_storage, decl_event, storage::child, ensure,
+	traits::Get, dispatch::Result,
+};
+use system::ensure_signed;
+use substrate_primitives::storage::well_known_keys::CHILD_STORAGE_KEY_PREFIX;
+use primitives::{Blake2Hasher, Hasher};
+
+// TODO: check if this is unnecessary dependency
+use parity_scale_codec::{Encode, Decode};
```

Comment:
It is

Reply:
ok, yes, it is used to store the struct in runtime storage